### ISSUE TITLE
Unable to signup (v. 1.0.2) 

### ISSUE THREAD

**Issue by HristovCodes** [Appollo-CLI/Appollo#19]
When I run `appollo signup`  I get the following error after submitting an email, username and password.

```
$ appollo signup
Email: *************@gmail.com
Username: *************
Password:
Repeat for confirmation:
Error: Server not available 
```

**Comment by maxcmoi89** [comment]
Can you try again ? I think you signed up exactly at the same moment the server was down due to the upgrade to Appollo v1.0.2

**Comment by HristovCodes** [comment]
It's still not working, I'll try again in 30 mins to an hour and I'll report back if I'm still encountering the issue, if everything is working fine I'll close it later.

**Comment by HristovCodes** [comment]
Update it still says 'Error: Server not available`

**Comment by maxcmoi89** [comment]
I seem to be having the problem too now... I will check tomorrow and get back to you.

**Comment by Daphneyong11** [comment]
Same problem, just now check on still the same thing, telling me server not available

**Comment by tchavetdeuse** [comment]
Hi, this is now fixed in version 1.0.3, sorry for the inconvenience.
